packages/ui/src/components/Card/_utils/utils.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { IconProps } from '@open-condo/icons'
+
+import { Space } from '../../Space'
+import { Typography } from '../../Typography'
+
+
+export type CardLinkType = {
+    href: string
+    label: string
+    PreIcon?: React.FC<IconProps>
+    AfterIcon?: React.FC<IconProps>
+}
+
+export const renderLink = (linkProps: CardLinkType) => {
+    const { PreIcon, AfterIcon, href, label } = linkProps
+
+    return (
+        <Space size={8} direction='horizontal' align='center'>
+            {PreIcon && <PreIcon size='small' />}
+            <Typography.Link href={href}>{label}</Typography.Link>
```

Comment:
Will this link be able to work as an SPA link (without reloading the page)? Or should it be like a default link?

Reply:
By default it's be like default link.
Added `LinkWrapper` prop to make it possible to be spa link (wrap in next/link from outside)